app/models/concerns/orchestration/dhcp.rb

```diff
@@ -120,7 +120,7 @@ def dhcp_attrs(record_mac)
       :ip => ip,
       :mac => record_mac,
       :proxy => subnet.dhcp_proxy,
-      :network => subnet.network,
+      :network => subnet.masked_subnet_address,
```

Comment:
This changes the data type from `String` to `IPAddr`. Is that handled correctly elsewhere or should it use a string here?
```suggestion
      :network => subnet.masked_subnet_address.to_s,
```

Reply:
Good point.